lib/global-styles.php

```diff
@@ -261,19 +261,38 @@ function gutenberg_experimental_global_styles_get_theme() {
 	return $theme_config;
 }
 
+/**
+ * Convert style property to its CSS name.
+ *
+ * @param string $style_property Style property name.
+ * @return string CSS property name.
+ */
+function gutenberg_experimental_global_styles_get_css_property( $style_property ) {
+	switch ( $style_property ) {
```

Comment:
I guess instead of having a switch we may have an array with the mappings:
array(
'backgroundColor' => ''background-color',
'fontSize' => 'font-size',
...
)